lib/facility_factory.rb

```diff
@@ -0,0 +1,25 @@
+<<<<<<< HEAD
```

Comment:
This merge conflict should be a relatively easier merge fix, so I'd encourage you to use the VSCode merge editing tool with this one